source/val/validate_image.cpp

```diff
@@ -1087,6 +1091,23 @@ spv_result_t ValidateSampledImage(ValidationState_t& _,
       }
     }
   }
+
+  const Instruction* ld_inst;
+  {
+    const spv_parsed_operand_t& topnd = inst->operand(2);
+    int t_idx = (int)inst->word(topnd.offset);
+    ld_inst = _.FindDef(t_idx);
+  }
+
+  if (ld_inst->opcode() == spv::Op::OpLoad) {
+    int texture_id;
+    const spv_parsed_operand_t& popnd =
+        ld_inst->operand(2);  // variable to load
+    texture_id = (int)ld_inst->word(popnd.offset);
```

Comment:
Prefer not mixing operand and word calls.

Reply:
Could you suggest the SPIRV-Tool's preferred way of accessing an operand's ID ?

Reply:
Thanks much, Alan. Fixed.